<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FormatDialog</class>
    <widget class="QDialog" name="FormatDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>720</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QTextEdit" name="info_edit">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" name="btn_layout" rowstretch="1,1" columnstretch="1,1">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item row="0" column="0" colspan="2">
                        <widget class="QToolButton" name="copy_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(143, 89, 2);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>copy to clip board</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QToolButton" name="save_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(32, 74, 135);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>save</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QToolButton" name="back_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(196, 160, 0);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>back</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
